npm/vite-dev-server/index-template.html

```diff
@@ -7,43 +7,14 @@
     <title>Components App</title>
   </head>
   <body>
-    <div id="app"></div>
+    <div id="__cy_root"></div>
     <script type="module">
-
-    function appendTargetIfNotExists (id, tag = 'div', parent = document.body) {
-      let node = document.getElementById(id)
-
-      if (!node) {
-        node = document.createElement(tag)
-        node.setAttribute('id', id)
-        parent.appendChild(node)
-      }
-
-      node.innerHTML = ''
-
-      return node
-    }
+    import '/@vite/client'
 
     const specPath = "{{{specPath}}}"
 
       let importsToLoad = [() => import(specPath).catch(e => {
-        // if the import failed, it might be because of dependencies 
-        // so we try a quick refresh just in case it is
-        
-        // Since vite does not work with IE we can use URLSearchParams without polyfill
-        const searchParams = new URLSearchParams(window.location.search);
```

Comment:
Why don't we need this anymore?

Reply:
vite is now conne tiong properly to the websockets, so it does all this on its own through the client ;)